Module.php

```diff
@@ -8,9 +8,9 @@ class Module
      */
     public function getAutoloaderConfig()
     {
-        return array('Zend\Loader\StandardAutoloader' => array('namespaces' => array(
+        return [\Zend\Loader\StandardAutoloader::class => ['namespaces' => [
             __NAMESPACE__ => __DIR__.'/src/',
```

Comment:
Ugh; need to update the `.php_cs` settings to remove the "remove spaces around concatenation operators" rule. I can do this on merge, though.